### PR TITLE
modified index file in root directory

### PR DESCRIPTION
**Pull request by targonskii**
Modified index file in About directory. Just for easy to read code.
Modified index file in root directory. Added 1 horizontal line, 2 paragraphs, 2 symbols, 2 emojis, 1 image, 1 list, 1 table and 1 form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,15 +1,166 @@
...
+                <br />
+                <label for="lever">Your programming level</label>
+                <br />
+                <input type="range" name="level" id="level" min="1" max="10" />
+                <br />
+                <br />
+                <label for="html">What does HTML stand for?</label>
+                <br />
+                <br />
+                <input type="checkbox" name="html" id=""/>Hyperlinks and Text Markup Language
```
Comment (ghost) [inline index.html]: Text for input should be wrapped on `label`.

File: index.html [hunk 2]
```diff
@@ -1,15 +1,166 @@
...
+                <br />
+                <input type="checkbox" name="html" />Home Tool Markup Language
```
Comment (ghost) [inline index.html]: See the previous comment.

File: index.html [hunk 3]
```diff
@@ -1,15 +1,166 @@
...
+                <br />
+                <input type="checkbox" name="html" />Hyper Text Markup Language
```
Comment (ghost) [inline index.html]: See the previous comment.

File: index.html [hunk 4]
```diff
@@ -1,15 +1,166 @@
...
+    <p><a href="./about/index.html">Об авторе</a></p>
+    <hr />
+    <p>По специальности я <mark>инженер</mark> в автомобильной сфере. 
+        Работаю в этой области <ins>более <sub>1</sub><sup>0</sup> лет</ins>.
+    </p>
+    <p>Но порой может наступить момент, когда перестаешь получать удовлетворение от проделанной 
+        работы и как говоpится <q>начинаешь искать новый вызов</q>.
+    </p>
+    <p>&#9875; &#9883; &iquest; &#x1f34c; &#x1f349;</p>
+    <p><img src="https://www.formula1.com/content/dam/fom-website/manual/Misc/2021preseason/FerrariLaunch/SF21-Profilo-DS.jpg"
```
Comment (ghost) [inline index.html]: `<p>` only for text.

File: index.html [hunk 5]
```diff
@@ -1,15 +1,166 @@
...
+                    <li>Тег для работы с изображениями.</li>
+                    <li>Теги для работы со списками.</li>
+                    <li>Теги для работы с таблицами.</li>
+                    <li>Теги для работы с формами.</li>
+                    <li>Div-блоки.</li>
+                </ul>
+            </li>
+        </ol>
+    </p>
+    <p>
```
Comment (ghost) [inline index.html]: Use `<p>` only for writing text.

File: index.html [hunk 6]
```diff
@@ -1,15 +1,166 @@
...
+    <p>По специальности я <mark>инженер</mark> в автомобильной сфере. 
+        Работаю в этой области <ins>более <sub>1</sub><sup>0</sup> лет</ins>.
+    </p>
+    <p>Но порой может наступить момент, когда перестаешь получать удовлетворение от проделанной 
+        работы и как говоpится <q>начинаешь искать новый вызов</q>.
+    </p>
+    <p>&#9875; &#9883; &iquest; &#x1f34c; &#x1f349;</p>
+    <p><img src="https://www.formula1.com/content/dam/fom-website/manual/Misc/2021preseason/FerrariLaunch/SF21-Profilo-DS.jpg" 
+        width="500" alt="ferrari_sf21"></p>
+    <p>
```
Comment (ghost) [inline index.html:26]: `<p>` only for text.

File: index.html [hunk 7]
```diff
@@ -1,15 +1,166 @@
...
+                <td>&lt;tbody&gt;</td>
+                <td>Groups the body content in a table</td>
+            </tr>
+            <tr>
+                <td>&lt;tfoot&gt;</td>
+                <td>Groups the footer content in a table</td>
+            </tr>
+        </table>
+    </p>
+    <p>
```
Comment (ghost) [inline index.html]: <p> only for text.

File: index.html (hunk 1 above)
Comment (targonskii) [reply index.html]: Fixed bug with wrapping

File: index.html (hunk 2 above)
Comment (targonskii) [reply index.html]: Fixed bug with wrapping

File: index.html (hunk 3 above)
Comment (targonskii) [reply index.html]: Fixed bug with wrapping

File: index.html (hunk 4 above)
Comment (targonskii) [reply index.html]: Fixed bug with emojis

File: index.html (hunk 5 above)
Comment (targonskii) [reply index.html]: Fixed it

File: index.html (hunk 6 above)
Comment (targonskii) [reply index.html:26]: Fixed bug for image

File: index.html (hunk 7 above)
Comment (targonskii) [reply index.html]: Fixed trouble with tag P for form.

**Pull request merged**